### PR TITLE
I added dropdown header now

### PR DESCRIPTION
**Pull request by sarath67**

### CODE REVIEW

File: src/components/HeaderDropdown/HeaderDropdown.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
```
Comment (Dheepan04) [inline src/components/HeaderDropdown/HeaderDropdown.jsx:1]: nice addition -good

File: src/components/HeaderDropdown/HeaderDropdown.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            <div>
+
+            </div>
+        </div>
+
+    </div>
+  )
+}
+
+export default HeaderDropdown
```
Comment (Dheepan04) [inline src/components/HeaderDropdown/HeaderDropdown.jsx:28]: why?

**Review by Dheepan04 — APPROVED** [review]

File: src/components/HeaderDropdown/HeaderDropdown.jsx (hunk 1 above)
Comment (sarath67) [reply src/components/HeaderDropdown/HeaderDropdown.jsx:1]: thanks dheepan

File: src/components/HeaderDropdown/HeaderDropdown.jsx (hunk 2 above)
Comment (sarath67) [reply src/components/HeaderDropdown/HeaderDropdown.jsx:28]: i think you are asking Why this export default , it needs to be included to be tracked to homepage(needs to be imported)

**Pull request merged**